cli/src/main/kotlin/com/malinskiy/marathon/cli/ApplicationView.kt

```diff
@@ -87,6 +91,8 @@ private fun execute(cliConfiguration: CliConfiguration) {
         }
     } finally {
         stopKoin()
-        bugsnagExceptionsReporter.end()
+        if (bugsnagExceptionsReporter != null) {
```

Comment:
This is in case we fail to parse the Marathon file, for example, where `bugsnagExceptionsReporter` isn't initialized yet and we don't want to call `bugsnagExceptionsReporter.end()`